stable/postgresql/templates/statefulset-slaves.yaml

```diff
@@ -203,6 +206,9 @@ spec:
       - name: data
         emptyDir: {}
       {{- end }}
+      {{- if .Values.slave.extraVolumes }}
+      {{ toYaml .Values.slave.extraVolumes | indent 6}}
```

Comment:
```suggestion
      {{- toYaml .Values.slave.extraVolumes | nindent 6}}
```

Reply:
I see that in one it is indent 6 and in the other indent 8. Make sure which the right indentation is